deploy/examples/nfs.yaml

```diff
@@ -83,6 +83,11 @@ spec:
     #         configMap:
     #           name: my-nfs-sssd-config
     #           defaultMode: 0600 # mode must be 0600
+    #     additionalFiles:
+    #       volumeSource:
```

Comment:
this isn't a list of `additionalFiles` here, probably needs the ` - `. Also needs a `subDirectory`?